source/process/documentation-UItext-guidelines.rst

```diff
@@ -0,0 +1,193 @@
+- Examples:
+
+  - *This message is too long. Shorten it to 120 characters.*
+  - *The passwords do not match.*
+
+
+Guidelines for UI elements
+---------------------------
+Use this table when writing the text for UI elements such as windows, dialog boxes, labels, and prompts.
+
+.. list-table::
+   :widths: 10 30 10 15 35
+   :header-rows: 1
+
+   * - Element
+     - Image 
+     - Capitalization
+     - Phrasing
+     - Examples     
+   * - Menu
+     - |menu.png|
+     - All words
+     - - Noun, noun phrase, or verb
+       - No punctuation
+       - Not more than three words
+     - - *Members*
+       - *Account Preferences*
+       - *Log Out*
+   * - Tooltip
+     - |tooltip.png|
+     - First word
+     - - Sentence fragment or sentence
+       - No punctuation
+       - Include articles (*a*, *an*, *the*)
+     - - *Start a Zoom meeting*
+       - *Flag for following up*
+       - *Remove from this list*
+   * - In-field text
+     - |field.png|  
+     - First word
+     - - Sentence fragment, sentence, or word
+       - No punctuation
+       - Include articles (*a*, *an*, *the*)
+     - - *Add a comment*
+       - *Search*
+   * - Action button
+     - |action.png|
+     - All words
+     - - Verb or verb phrase
+       - No articles (*a*, *an*, *the*)
+       - Exceptions: *OK*, *Yes*, *No*
+     - - *Add Comment*
+       - *Edit*
+   * - Label before a UI element
+     - |label_before.png|
+     - First word
+     - - Noun, verb, or sentence fragment
+       - End with a colon
+       - Include articles (*a*, *an*, *the*)
+     - - *Sign in with:*
+       - *Other words, separated by commas:*
+   * - Label after a UI element
+     - |label_after.png|
+     - First word
+     - - Noun, verb, or sentence fragment
+       - No punctuation
+     - - *Channels grouped by type*
+       - *Alphabetically*
+   * - Help text
+     - |help.png|
+     - First word
+     - Complete sentences, with punctuation
+     - - *You can add 20 more people.*
+       - *People are invited automatically to join the channel.*
+   * - Title
+     - |title.png|
+     - All words
+     - - Sentence fragment or sentence
+       - No punctuation
+     - - *Notification Preferences for Channel*
+       - *Contributors*
+   
+Cheatsheet: UI text
+-----------------------
+|cheatsheet.png|
+
+.. |menu.png| image:: ./images/menu.png
+  :alt: menu
+.. |tooltip.png| image:: ./images/tooltip.png
+  :alt: tooltip
+.. |field.png| image:: ./images/field.png
+  :alt: in-field text
+.. |action.png| image:: ./images/action.png
+  :alt: action button
+.. |label_before.png| image:: ./images/label_before.png
+  :alt: labels before a UI element
+.. |label_after.png| image:: ./images/label_after.png
+  :alt: labels after a UI element
+.. |help.png| image:: ./images/help.png
+  :alt: help text
+.. |title.png| image:: ./images/title.png
+  :alt: title
+.. |cheatsheet.png| image:: ./images/cheatsheet.png
+  :alt: cheatsheet for punctuation and capitalization
```

Comment:
minor - the tooltip textbox has a typo in the screenshot: `puntuation` --> `punctuation` . Since it's a screenshot, it may not be easy to resolve so I wouldn't worry if not easy to fix.

Reply:
I created the image in Excel, so it's still editable easily. Fixed.